libmediapipe_c has a duplicated `lib-` prefix on KDE neon

Distro: KDE Neon 20.04

Hi there,

whenever I try to run Vignette I get the following errors:
```
robert@robert-pc /m/r/F/P/Vignette> ./Vignette
[runtime] 2022-01-01 14:14:12 [verbose]: Connected MIDI device: Midi Through Port-0
[runtime] 2022-01-01 14:14:12 [verbose]: GL Initialized
[runtime] 2022-01-01 14:14:12 [verbose]: GL Version:                 4.6.0 NVIDIA 470.86
[runtime] 2022-01-01 14:14:12 [verbose]: GL Renderer:                NVIDIA GeForce RTX 2060/PCIe/SSE2
[runtime] 2022-01-01 14:14:12 [verbose]: GL Shader Language version: 4.60 NVIDIA
[runtime] 2022-01-01 14:14:12 [verbose]: GL Vendor:                  NVIDIA Corporation
[runtime] 2022-01-01 14:14:12 [verbose]: GL Extensions:              GL_AMD_multi_draw_indirect GL_AMD_seamless_cubemap_per_texture GL_AMD_vertex_shader_viewport_index GL_AMD_vertex_shader_layer GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_bindless_texture GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_color_buffer_float GL_ARB_compatibility GL_ARB_compressed_texture_pixel_storage GL_ARB_conservative_depth GL_ARB_compute_shader GL_ARB_compute_variable_group_size GL_ARB_conditional_render_inverted GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_depth_texture GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_indirect GL_ARB_draw_elements_base_vertex GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_ES2_compatibility GL_ARB_ES3_compatibility GL_ARB_ES3_1_compatibility GL_ARB_ES3_2_compatibility GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_program GL_ARB_fragment_program_shadow GL_ARB_fragment_shader GL_ARB_fragment_shader_interlock GL_ARB_framebuffer_no_attachments GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_geometry_shader4 GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gl_spirv GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader_int64 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_imaging GL_ARB_indirect_parameters GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_multisample GL_ARB_multitexture GL_ARB_occlusion_query GL_ARB_occlusion_query2 GL_ARB_parallel_shader_compile GL_ARB_pipeline_statistics_query GL_ARB_pixel_buffer_object GL_ARB_point_parameters GL_ARB_point_sprite GL_ARB_polygon_offset_clamp GL_ARB_post_depth_coverage GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_query_buffer_object GL_ARB_robust_buffer_access_behavior GL_ARB_robustness GL_ARB_sample_locations GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_atomic_counter_ops GL_ARB_shader_atomic_counters GL_ARB_shader_ballot GL_ARB_shader_bit_encoding GL_ARB_shader_clock GL_ARB_shader_draw_parameters GL_ARB_shader_group_vote GL_ARB_shader_image_load_store GL_ARB_shader_image_size GL_ARB_shader_objects GL_ARB_shader_precision GL_ARB_shader_storage_buffer_object GL_ARB_shader_subroutine GL_ARB_shader_texture_image_samples GL_ARB_shader_texture_lod GL_ARB_shading_language_100 GL_ARB_shader_viewport_layer_array GL_ARB_shading_language_420pack GL_ARB_shading_language_include GL_ARB_shading_language_packing GL_ARB_shadow GL_ARB_sparse_buffer GL_ARB_sparse_texture GL_ARB_sparse_texture2 GL_ARB_sparse_texture_clamp GL_ARB_spirv_extensions GL_ARB_stencil_texturing GL_ARB_sync GL_ARB_tessellation_shader GL_ARB_texture_barrier GL_ARB_texture_border_clamp GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression GL_ARB_texture_compression_bptc GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map GL_ARB_texture_cube_map_array GL_ARB_texture_env_add GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_filter_anisotropic GL_ARB_texture_filter_minmax GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_mirrored_repeat GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_query_lod GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_stencil8 GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_transform_feedback_overflow_query GL_ARB_transpose_matrix GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_64bit GL_ARB_vertex_attrib_binding GL_ARB_vertex_buffer_object GL_ARB_vertex_program GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ARB_window_pos GL_ATI_draw_buffers GL_ATI_texture_float GL_ATI_texture_mirror_once GL_S3_s3tc GL_EXT_texture_env_add GL_EXT_abgr GL_EXT_bgra GL_EXT_bindable_uniform GL_EXT_blend_color GL_EXT_blend_equation_separate GL_EXT_blend_func_separate GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_compiled_vertex_array GL_EXT_Cg_shader GL_EXT_depth_bounds_test GL_EXT_direct_state_access GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_draw_range_elements GL_EXT_fog_coord GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXTX_framebuffer_mixed_formats GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_object GL_EXT_framebuffer_sRGB GL_EXT_geometry_shader4 GL_EXT_gpu_program_parameters GL_EXT_gpu_shader4 GL_EXT_multi_draw_arrays GL_EXT_multiview_texture_multisample GL_EXT_multiview_timer_query GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_packed_pixels GL_EXT_pixel_buffer_object GL_EXT_point_parameters GL_EXT_polygon_offset_clamp GL_EXT_post_depth_coverage GL_EXT_provoking_vertex GL_EXT_raster_multisample GL_EXT_rescale_normal GL_EXT_secondary_color GL_EXT_separate_shader_objects GL_EXT_separate_specular_color GL_EXT_shader_image_load_formatted GL_EXT_shader_image_load_store GL_EXT_shader_integer_mix GL_EXT_shadow_funcs GL_EXT_sparse_texture2 GL_EXT_stencil_two_side GL_EXT_stencil_wrap GL_EXT_texture3D GL_EXT_texture_array GL_EXT_texture_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_latc GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_cube_map GL_EXT_texture_edge_clamp GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_EXT_texture_filter_anisotropic GL_EXT_texture_filter_minmax GL_EXT_texture_integer GL_EXT_texture_lod GL_EXT_texture_lod_bias GL_EXT_texture_mirror_clamp GL_EXT_texture_object GL_EXT_texture_shadow_lod GL_EXT_texture_shared_exponent GL_EXT_texture_sRGB GL_EXT_texture_sRGB_R8 GL_EXT_texture_sRGB_decode GL_EXT_texture_storage GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback2 GL_EXT_vertex_array GL_EXT_vertex_array_bgra GL_EXT_vertex_attrib_64bit GL_EXT_window_rectangles GL_EXT_x11_sync_object GL_EXT_import_sync_object GL_NV_robustness_video_memory_purge GL_IBM_rasterpos_clip GL_IBM_texture_mirrored_repeat GL_KHR_context_flush_control GL_KHR_debug GL_EXT_memory_object GL_EXT_memory_object_fd GL_NV_memory_object_sparse GL_KHR_parallel_shader_compile GL_KHR_no_error GL_KHR_robust_buffer_access_behavior GL_KHR_robustness GL_EXT_semaphore GL_EXT_semaphore_fd GL_NV_timeline_semaphore GL_KHR_shader_subgroup GL_KTX_buffer_region GL_NV_alpha_to_coverage_dither_control GL_NV_bindless_multi_draw_indirect GL_NV_bindless_multi_draw_indirect_count GL_NV_bindless_texture GL_NV_blend_equation_advanced GL_NV_blend_equation_advanced_coherent GL_NVX_blend_equation_advanced_multi_draw_buffers GL_NV_blend_minmax_factor GL_NV_blend_square GL_NV_clip_space_w_scaling GL_NV_command_list GL_NV_compute_program5 GL_NV_compute_shader_derivatives GL_NV_conditional_render GL_NV_conservative_raster GL_NV_conservative_raster_dilate GL_NV_conservative_raster_pre_snap GL_NV_conservative_raster_pre_snap_triangles GL_NV_conservative_raster_underestimation GL_NV_copy_depth_to_color GL_NV_copy_image GL_NV_depth_buffer_float GL_NV_depth_clamp GL_NV_draw_texture GL_NV_draw_vulkan_image GL_NV_ES1_1_compatibility GL_NV_ES3_1_compatibility GL_NV_explicit_multisample GL_NV_feature_query GL_NV_fence GL_NV_fill_rectangle GL_NV_float_buffer GL_NV_fog_distance GL_NV_fragment_coverage_to_color GL_NV_fragment_program GL_NV_fragment_program_option GL_NV_fragment_program2 GL_NV_fragment_shader_barycentric GL_NV_fragment_shader_interlock GL_NV_framebuffer_mixed_samples GL_NV_framebuffer_multisample_coverage GL_NV_geometry_shader4 GL_NV_geometry_shader_passthrough GL_NV_gpu_program4 GL_NV_internalformat_sample_query GL_NV_gpu_program4_1 GL_NV_gpu_program5 GL_NV_gpu_program5_mem_extended GL_NV_gpu_program_fp64 GL_NV_gpu_shader5 GL_NV_half_float GL_NV_light_max_exponent GL_NV_memory_attachment GL_NV_mesh_shader GL_NV_multisample_coverage GL_NV_multisample_filter_hint GL_NV_occlusion_query GL_NV_packed_depth_stencil GL_NV_parameter_buffer_object GL_NV_parameter_buffer_object2 GL_NV_path_rendering GL_NV_path_rendering_shared_edge GL_NV_pixel_data_range GL_NV_point_sprite GL_NV_primitive_restart GL_NV_query_resource GL_NV_query_resource_tag GL_NV_register_combiners GL_NV_register_combiners2 GL_NV_representative_fragment_test GL_NV_sample_locations GL_NV_sample_mask_override_coverage GL_NV_scissor_exclusive GL_NV_shader_atomic_counters GL_NV_shader_atomic_float GL_NV_shader_atomic_float64 GL_NV_shader_atomic_fp16_vector GL_NV_shader_atomic_int64 GL_NV_shader_buffer_load GL_NV_shader_storage_buffer_object GL_NV_shader_subgroup_partitioned GL_NV_shader_texture_footprint GL_NV_shading_rate_image GL_NV_stereo_view_rendering GL_NV_texgen_reflection GL_NV_texture_barrier GL_NV_texture_compression_vtc GL_NV_texture_env_combine4 GL_NV_texture_multisample GL_NV_texture_rectangle GL_NV_texture_rectangle_compressed GL_NV_texture_shader GL_NV_texture_shader2 GL_NV_texture_shader3 GL_NV_transform_feedback GL_NV_transform_feedback2 GL_NV_uniform_buffer_unified_memory GL_NV_vdpau_interop GL_NV_vdpau_interop2 GL_NV_vertex_array_range GL_NV_vertex_array_range2 GL_NV_vertex_attrib_integer_64bit GL_NV_vertex_buffer_unified_memory GL_NV_vertex_program GL_NV_vertex_program1_1 GL_NV_vertex_program2 GL_NV_vertex_program2_option GL_NV_vertex_program3 GL_NV_viewport_array2 GL_NV_viewport_swizzle GL_NVX_conditional_render GL_NV_gpu_multicast GL_NVX_progress_fence GL_NVX_gpu_memory_info GL_NVX_nvenc_interop GL_NV_shader_thread_group GL_NV_shader_thread_shuffle GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_OVR_multiview GL_OVR_multiview2 GL_SGIS_generate_mipmap GL_SGIS_texture_lod GL_SGIX_depth_texture GL_SGIX_shadow GL_SUN_slice_accum
[runtime] 2022-01-01 14:14:12 [verbose]: Host execution state changed to Running
[runtime] 2022-01-01 14:14:12 [verbose]: Execution mode changed to MultiThreaded
[runtime] 2022-01-01 14:14:12 [debug]: Loading Font FontAwesome-Solid...
[runtime] 2022-01-01 14:14:12 [debug]: Loading Font Roboto-Regular...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font FontAwesome-Solid!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font FontAwesome-Regular...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font FontAwesome-Regular!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font FontAwesome-Brands...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font FontAwesome-Brands!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-Italic...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font Roboto-Regular!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font Roboto-RegularItalic...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-Italic!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-Bold...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-Bold!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-BoldItalic...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font Roboto-RegularItalic!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font Roboto-Bold...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font Roboto-Bold!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font Roboto-BoldItalic...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-BoldItalic!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-Black...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-Black!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-BlackItalic...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font Roboto-BoldItalic!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font RobotoCondensed-Regular...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-BlackItalic!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-Light...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font RobotoCondensed-Regular!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font RobotoCondensed-Bold...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-Light!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-LightItalic...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-LightItalic!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-SemiBold...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font RobotoCondensed-Bold!
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-SemiBold!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-SemiBoldItalic...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-SemiBoldItalic!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-SemiLight...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-SemiLight!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeUI-SemiLightItalic...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeUI-SemiLightItalic!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font Spartan-Bold...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font Spartan-Bold!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font SegoeFluent-Regular...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font SegoeFluent-Regular!
[runtime] 2022-01-01 14:14:13 [debug]: Loading Font Vignette...
[runtime] 2022-01-01 14:14:13 [debug]: Loaded Font Vignette!
[runtime] 2022-01-01 14:14:13 [error]: An unhandled error has occurred.
[runtime] 2022-01-01 14:14:13 [error]: System.DllNotFoundException: Unable to load shared library 'libmediapipe_c' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: liblibmediapipe_c: cannot open shared object file: No such file or directory
[runtime] 2022-01-01 14:14:13 [error]: at Akihabara.Native.Framework.UnsafeNativeMethods.mp_CalculatorGraph__PKc(String textFormatConfig, IntPtr& graph)
[runtime] 2022-01-01 14:14:13 [error]: at Akihabara.Framework.CalculatorGraph..ctor(String textFormatConfig)
[runtime] 2022-01-01 14:14:13 [error]: at Vignette.Game.Tracking.TrackingComponent.load(IBindable`1 camera) in /home/runner/work/vignette/vignette/Vignette.Game/Tracking/TrackingComponent.cs:line 65
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.AddInternal(Drawable drawable)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.Container`1.AddInternal(Drawable drawable)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.Container`1.Add(T drawable)
[runtime] 2022-01-01 14:14:13 [error]: at Vignette.Game.VignetteGameBase.load() in /home/runner/work/vignette/vignette/Vignette.Game/VignetteGameBase.cs:line 134
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
[runtime] 2022-01-01 14:14:13 [error]: --- End of stack trace from previous location ---
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Platform.GameHost.bootstrapSceneGraph(Game game)
[runtime] 2022-01-01 14:14:13 [error]: at osu.Framework.Platform.GameHost.Run(Game game)
[runtime] 2022-01-01 14:14:13 [error]: at Vignette.Desktop.Program.Main(String[] args) in /home/runner/work/vignette/vignette/Vignette.Desktop/Program.cs:line 13
Unhandled exception. System.DllNotFoundException: Unable to load shared library 'libmediapipe_c' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: liblibmediapipe_c: cannot open shared object file: No such file or directory
   at Akihabara.Native.Framework.UnsafeNativeMethods.mp_CalculatorGraph__PKc(String textFormatConfig, IntPtr& graph)
   at Akihabara.Framework.CalculatorGraph..ctor(String textFormatConfig)
   at Vignette.Game.Tracking.TrackingComponent.load(IBindable`1 camera) in /home/runner/work/vignette/vignette/Vignette.Game/Tracking/TrackingComponent.cs:line 65
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
   at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
   at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
   at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
   at osu.Framework.Graphics.Containers.CompositeDrawable.AddInternal(Drawable drawable)
   at osu.Framework.Graphics.Containers.Container`1.AddInternal(Drawable drawable)
   at osu.Framework.Graphics.Containers.Container`1.Add(T drawable)
   at Vignette.Game.VignetteGameBase.load() in /home/runner/work/vignette/vignette/Vignette.Game/VignetteGameBase.cs:line 134
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
   at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
   at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
   at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Graphics.Containers.CompositeDrawable.loadChild(Drawable child)
   at osu.Framework.Graphics.Containers.CompositeDrawable.load(ShaderManager shaders, Nullable`1 cancellation)
--- End of stack trace from previous location ---
   at osu.Framework.Allocation.BackgroundDependencyLoaderAttribute.<>c__DisplayClass6_0.<CreateActivator>b__4(Object target, IReadOnlyDependencyContainer dc)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyActivator.Activate(Object obj, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Allocation.DependencyContainer.Inject[T](T instance)
   at osu.Framework.Graphics.Drawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Containers.CompositeDrawable.InjectDependencies(IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies)
   at osu.Framework.Graphics.Drawable.Load(IFrameBasedClock clock, IReadOnlyDependencyContainer dependencies, Boolean isDirectAsyncContext)
   at osu.Framework.Platform.GameHost.bootstrapSceneGraph(Game game)
   at osu.Framework.Platform.GameHost.Run(Game game)
   at Vignette.Desktop.Program.Main(String[] args) in /home/runner/work/vignette/vignette/Vignette.Desktop/Program.cs:line 13
fish: Job 1, './Vignette' terminated by signal SIGABRT (Abort)
robert@robert-pc /m/r/F/P/Vignette [SIGABRT]>
```

Especially the "liblibmediapipe_c" in `In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: liblibmediapipe_c` does not seem right.

If there is anything else I need to provide please lmk :D

I have cloned the repo, checked out the branch and installed Python 3.9, .NET SDK 5.0, GCC-9 and Bazel, but now I am getting the following errors:
```
robert@robert-pc /m/r/F/P/Akihabara ((33445796…)) [1]> python build.py build --desktop cpu -vv
INFO (build.py): Building native libraries for Linux Desktop...
DEBUG (build.py): Running `bazel build -c opt --verbose_failures --define MEDIAPIPE_DISABLE_GPU=1 --@opencv//:switch=local //mediapipe_api:mediapipe_desktop`
WARNING: Download from http://mirror.tensorflow.org/github.com/bazelbuild/rules_closure/archive/cf1e44edb908e9616030cc83d085989b8e6cd6df.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
WARNING: Download from http://mirror.tensorflow.org/github.com/tensorflow/runtime/archive/44c890cd46fd9aad2b3bf4b13c5f36d8cff795c3.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'com_google_absl' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'com_google_protobuf' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'com_google_googletest' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'com_github_gflags_gflags' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'rules_python' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'build_bazel_rules_apple' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'build_bazel_rules_swift' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'build_bazel_apple_support' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'bazel_skylib' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/org_tensorflow/third_party/repo.bzl:108:14: 
Warning: skipping import of repository 'pybind11' because it already exists.
DEBUG: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/tf_runtime/third_party/cuda/dependencies.bzl:51:10: The following command will download NVIDIA proprietary software. By using the software you agree to comply with the terms of the license agreement that accompanies the software. If you do not agree to the terms of the license agreement, do not use the software.
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1564776078 -0400"
DEBUG: Repository rules_python instantiated at:
  /media/robert/Files/Programs/Akihabara/WORKSPACE:35:23: in <toplevel>
  /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/rules_pkg/deps.bzl:33:10: in rules_pkg_dependencies
  /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/bazel_tools/tools/build_defs/repo/utils.bzl:201:18: in maybe
Repository rule git_repository defined at:
  /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/bazel_tools/tools/build_defs/repo/git.bzl:199:33: in <toplevel>
DEBUG: Rule 'rules_cc' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3839996049629e6377abdfd04681ddeeb0cc3db13b9d2ff81bf46700cb4529f7"
DEBUG: Repository rules_cc instantiated at:
  /media/robert/Files/Programs/Akihabara/WORKSPACE:80:13: in <toplevel>
Repository rule http_archive defined at:
  /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
WARNING: Download from https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/ruy/archive/d37128311b445e758136b8602d1bbd2a755e115d.zip failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
INFO: Analyzed target //mediapipe_api:mediapipe_desktop (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
ERROR: /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/external/com_google_mediapipe/mediapipe/modules/objectron/calculators/BUILD:176:11: Compiling mediapipe/modules/objectron/calculators/tensor_util.cc failed: (Exit 1): gcc failed: error executing command 
  (cd /home/robert/.cache/bazel/_bazel_robert/17a4b612af0f800a11566ad528ec570f/sandbox/linux-sandbox/748/execroot/mediapipe_api && \
  exec env - \
    PATH=/home/robert/.local/share/nvm/v16.13.1/bin:/opt/lampp:/home/robert/spicetify-cli:/home/robert/.nvm:/home/robert/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin \
    PWD=/proc/self/cwd \
  /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g0 -O2 '-D_FORTIFY_SOURCE=1' -DNDEBUG -ffunction-sections -fdata-sections '-std=c++0x' -MD -MF bazel-out/k8-opt/bin/external/com_google_mediapipe/mediapipe/modules/objectron/calculators/_objs/tensor_util/tensor_util.pic.d '-frandom-seed=bazel-out/k8-opt/bin/external/com_google_mediapipe/mediapipe/modules/objectron/calculators/_objs/tensor_util/tensor_util.pic.o' -fPIC '-DMEDIAPIPE_DISABLE_GPU=1' -iquote external/com_google_mediapipe -iquote bazel-out/k8-opt/bin/external/com_google_mediapipe -iquote external/com_google_absl -iquote bazel-out/k8-opt/bin/external/com_google_absl -iquote external/com_github_glog_glog -iquote bazel-out/k8-opt/bin/external/com_github_glog_glog -iquote external/com_github_gflags_gflags -iquote bazel-out/k8-opt/bin/external/com_github_gflags_gflags -iquote external/linux_opencv -iquote bazel-out/k8-opt/bin/external/linux_opencv -iquote external/org_tensorflow -iquote bazel-out/k8-opt/bin/external/org_tensorflow -iquote external/flatbuffers -iquote bazel-out/k8-opt/bin/external/flatbuffers -iquote external/ruy -iquote bazel-out/k8-opt/bin/external/ruy -Ibazel-out/k8-opt/bin/external/com_github_glog_glog/_virtual_includes/default_glog_headers -Ibazel-out/k8-opt/bin/external/com_github_gflags_gflags/_virtual_includes/gflags -Ibazel-out/k8-opt/bin/external/flatbuffers/_virtual_includes/runtime_cc -Ibazel-out/k8-opt/bin/external/flatbuffers/_virtual_includes/flatbuffers -Ibazel-out/k8-opt/bin/external/flatbuffers/src/_virtual_includes/flatbuffers -isystem external/linux_opencv/include/opencv4 -isystem bazel-out/k8-opt/bin/external/linux_opencv/include/opencv4 -isystem external/org_tensorflow/tensorflow/lite/schema -isystem bazel-out/k8-opt/bin/external/org_tensorflow/tensorflow/lite/schema -isystem external/org_tensorflow/tensorflow/lite/experimental/acceleration/configuration -isystem bazel-out/k8-opt/bin/external/org_tensorflow/tensorflow/lite/experimental/acceleration/configuration -w '-std=c++17' -fno-canonical-system-headers -Wno-builtin-macro-redefined '-D__DATE__="redacted"' '-D__TIMESTAMP__="redacted"' '-D__TIME__="redacted"' -c external/com_google_mediapipe/mediapipe/modules/objectron/calculators/tensor_util.cc -o bazel-out/k8-opt/bin/external/com_google_mediapipe/mediapipe/modules/objectron/calculators/_objs/tensor_util/tensor_util.pic.o)
Execution platform: @local_execution_config_platform//:platform

Use --sandbox_debug to see verbose messages from the sandbox
In file included from external/com_google_mediapipe/mediapipe/modules/objectron/calculators/tensor_util.h:19,
                 from external/com_google_mediapipe/mediapipe/modules/objectron/calculators/tensor_util.cc:15:
external/com_google_mediapipe/mediapipe/framework/port/opencv_core_inc.h:18:10: fatal error: opencv2/core/version.hpp: No such file or directory
   18 | #include <opencv2/core/version.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Target //mediapipe_api:mediapipe_desktop failed to build
INFO: Elapsed time: 0.412s, Critical Path: 0.22s
INFO: 129 processes: 129 internal.
FAILED: Build did NOT complete successfully
Traceback (most recent call last):
  File "/media/robert/Files/Programs/Akihabara/build.py", line 436, in <module>
    Argument().command().run()
  File "/media/robert/Files/Programs/Akihabara/build.py", line 139, in run
    self._run_command(self._build_desktop_commands())
  File "/media/robert/Files/Programs/Akihabara/build.py", line 50, in _run_command
    return subprocess.run(' '.join(command_list), check=True, shell=shell)
  File "/usr/lib/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'bazel build -c opt --verbose_failures --define MEDIAPIPE_DISABLE_GPU=1 --@opencv//:switch=local //mediapipe_api:mediapipe_desktop' returned non-zero exit status 1.
robert@robert-pc /m/r/F/P/Akihabara ((33445796…)) [1]>
```

--- a/mediapipe_api/BUILD
+++ b/mediapipe_api/BUILD
@@ -60,7 +60,7 @@ cc_binary(
 )
 
 cc_binary(
-    name = "libmediapipe_c.dll",
+    name = "mediapipe_c.dll",
     deps = [":mediapipe_c"],
     linkshared = True,
     linkstatic = True,

--- a/mediapipe_api/BUILD
+++ b/mediapipe_api/BUILD
@@ -71,7 +71,7 @@ alias(
     name = "libmediapipe_c",
     actual = select({
         "@com_google_mediapipe//mediapipe:macos": ":libmediapipe_c.dylib",
-        "@com_google_mediapipe//mediapipe:windows": ":libmediapipe_c.dll",
+        "@com_google_mediapipe//mediapipe:windows": ":mediapipe_c.dll",
         "//conditions:default": ":libmediapipe_c.so",
     }),
 )

--- a/src/Akihabara/Native/NativeMethods.cs
+++ b/src/Akihabara/Native/NativeMethods.cs
@@ -13,6 +13,6 @@ namespace Akihabara.Native
         // variant we're basing off, which you can find here: https://git.io/Jc3y1
         //
         // However, since we're targeting PC for now, we'll go for mediapipe_c for now.
-        internal const string MediaPipeLibrary = "libmediapipe_c";
+        internal const string MediaPipeLibrary = "mediapipe_c";
     }
 }

